membership-attribute-service/app/monitoring/SentryLogging.scala

```diff
@@ -1,43 +1,25 @@
 package monitoring
 
-import ch.qos.logback.classic.filter.ThresholdFilter
-import ch.qos.logback.classic.{Logger, LoggerContext}
-import com.getsentry.raven.RavenFactory
-import com.getsentry.raven.logback.SentryAppender
+import io.sentry.Sentry
 import configuration.Config
-import org.slf4j.Logger.ROOT_LOGGER_NAME
-import org.slf4j.LoggerFactory
+import scala.collection.JavaConversions._
+import scala.util.{Failure, Success, Try}
 
 object SentryLogging {
 
   val UserIdentityId = "userIdentityId"
   val UserGoogleId = "userGoogleId"
-  val AllMDCTags = Seq(UserIdentityId, UserGoogleId)
-
-  val RavenOpt = Config.sentryDsn.map(RavenFactory.ravenInstance).toOption
+  val AllMDCTags = Set(UserIdentityId, UserGoogleId)
```

Comment:
I know you didn't add this, but I don't think we want to send this to Sentry anymore, as it will be considered personal data under the GDPR